README.md

```diff
@@ -57,7 +57,23 @@ self.addEventListener('periodicsync', event => {
 
 # Security and Privacy
 
-TODO
+One-shot Background Sync allows the web page’s logic to live a little longer (service worker execution time) after the page has been closed by the user. Periodic Background Sync extends it to potentially infinite time thereafter. Here are some privacy concerns:
+
+* There is no notification to the user.
+* A periodic sync can be enabled while the user is connected to one network, and the sync event
+can be fired later when they're connected to another network. This can cause inadvertent leakage 
+of browsing history on an unintended network. This concern applies to other background tasks such as one-shot Sync and Background Fetch as well.
+* Location tracking: The user’s IP address can be revealed to the website every time the periodic task is run. This risk is also present with any tasks run in a service worker, but Periodic Sync allows persistent tracking. Note that this risk is also present with Push messages,
+but they require explicit user permission.
+
+The following steps should be taken by the user agent to mitigate these privacy concerns:
+* Limit the number of times the periodic tasks are run to a few times a day.
+* Make the frequency of these tasks inversely proportional to the user's engagement with a site. Suspend registrations for sites with which the user isn't engaging anymore.
+* Gate the API behind a user permission, or make it available contingent to the availability of a PWA for the origin on the device.
+* The existing UI controls that let users disable one-shot Background Sync should also disable
+Periodic Background Sync.
```

Comment:
This is also a Chrome decision. I think it would be good to separate the two entirely.

Reply:
It's quite hard to talk about mitigation of privacy concerns without providing some recommendations. Does it make sense to not discuss mitigations at all?